etc/kube/push-to-minikube.sh

```diff
@@ -6,6 +6,11 @@ if [[ $# -ne 1 ]]; then
   echo "error: need the name of the docker image to push"
 fi
 
+if [ -f /TESTFASTER_PREWARM_COMPLETE ]; then
```

Comment:
Does testfaster use Minikube? Again, not a problem, but I guess I'd expect the `if` statement below to do the same job. Happy to leave this in if it doesn't.

Reply:
it does, and i'm not sure why it wasn't working. i can try and figure that out and remove this as a followup.